src/components/commentForm/index.js

```diff
@@ -0,0 +1,90 @@
+import React, {Component} from 'react';
+import {connect} from "react-redux";
+
+
+import "./index.scss";
+class CommentForm extends Component{
+
+    constructor(props){
+        super(props);
+        this.state = {
+            commentInput: "",
+        }
+    }
+
+    componentDidMount(){
+        // Add comments from the right array. 
+    }
+
+
+    handleForm = event => {
+        event.preventDefault();
+    }
+    handleSubmit = event => {
+    }
+
+    handleClear = event =>  {
+        document.querySelector("textarea").value = "";
```

Comment:
You can also add a value on the `textarea` that is linked to the `commentInput` on the local state. Then when you want to reset the field you can use `setState` and set the `commentInput` to empty string.